k9mail/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -401,9 +401,17 @@ public void onRecipientRemove(Recipient currentRecipient) {
     }
 
     @Override
-    public void onRecipientChange(Recipient currentRecipient, Recipient alternateAddress) {
+    public void onRecipientChange(Recipient recipientToReplace, Recipient alternateAddress) {
         alternatesPopup.dismiss();
 
+        List<Recipient> currentRecipients = getObjects();
+        int indexOfRecipient = currentRecipients.indexOf(alternateAddress);
```

Comment:
You're right, and recipientToReplace isn't used anywhere either. Seems I miscommitted this. I'll test again to make sure this behaves right.